packages/protocol-contracts/contracts/ZetaInteractor.sol

```diff
@@ -1,12 +1,12 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.7;
 
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "./openzeppelin/Ownable2Step.sol";
```

Comment:
Why not from the dependencies?

Reply:
auditors ask us to include it but is not yet in the pkg, it's only in the OZ repo